samples/EarlyInitializationSample/Program.cs

```diff
@@ -21,6 +21,7 @@ public static int Main(string[] args)
             Log.Logger = new LoggerConfiguration()
                 .ReadFrom.Configuration(Configuration)
                 .Enrich.FromLogContext()
+                .WriteTo.Debug()
```

Comment:
I like this, there are lots of questions when folks are getting started that this may help.

Reply:
:+1: